components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -124,7 +132,44 @@ define([
             var exportType = $$('input:checked[type=radio][name="export-type"]')[0].value;
             //console.log("Import type: " + exportType);
 
-            if (exportType == "simpleJSON") {
+            if (exportType == "image") {
+                var imageType = $$('input:checked[type=radio][name="image-options"]')[0].value;
+
+				var _bbox = $('canvas').down().getBBox();
+                //set white background
+                var svgEl = $$("#canvas svg")[0];
+                var background = $$('.panning-background')[0];
+                svgEl.insertBefore(
+                    new Element('rect', {"id": "white-bbox-background",
+                                         "width": _bbox.width,
+                                         "height": _bbox.height,
+                                         "x": _bbox.x,
+                                         "y": _bbox.y,
+                                         "fill": "white"}),
+                                svgEl.firstChild
```

Comment:
but shouldn't it align with "new" since it "new Element" is first parameter, svgEl is second to the same function?